package/kernel/linux/modules/fs.mk

```diff
@@ -232,6 +232,8 @@ define KernelPackage/fs-ext4
   KCONFIG:= \
 	CONFIG_EXT4_FS \
 	CONFIG_EXT4_ENCRYPTION=n \
+	CONFIG_EXT4_FS_POSIX_ACL=y \
+	CONFIG_EXT4_FS_SECURITY=y \
```

Comment:
Some targets are building `CONFIG_EXT4_FS` into the kernel. There this package is not build. 
I think it would be better to move these two options to `target/linux/generic/config-*`. 

How much bigger is the kernel getting with these two options and their dependencies?

Reply:
> How much bigger is the kernel getting with these two options and their dependencies?

differences are minimal
normal
```
ls -l --block-size=K
12950K Aug 25 16:44  openwrt-x86-64-generic-ext4-combined-efi.img.gz
12754K Aug 25 16:44  openwrt-x86-64-generic-ext4-combined.img.gz
5663K Aug 25 16:44  openwrt-x86-64-generic-ext4-rootfs.img.gz
6701K Aug 25 16:43  openwrt-x86-64-generic-kernel.bin
```
with these two options
```
ls -l --block-size=K
12947K Aug 25 17:18 openwrt-x86-64-generic-ext4-combined-efi.img.gz
12759K Aug 25 17:18 openwrt-x86-64-generic-ext4-combined.img.gz
5663K Aug 25 17:18 openwrt-x86-64-generic-ext4-rootfs.img.gz
6705K Aug 25 17:18 openwrt-x86-64-generic-kernel.bin
```



> Some targets are building `CONFIG_EXT4_FS` into the kernel. There this package is not build.
> I think it would be better to move these two options to `target/linux/generic/config-*`.

I push changes following your suggestion, also deactivated  those options from targets where are enabled.